es-ds-docs/pages/molecules/checkbox.vue

```diff
@@ -0,0 +1,180 @@
+<script setup>
+import { ref } from 'vue'
+
+const apple = ref(false);
+const banana = ref(true);
+const cherry = ref(false);
+const selectAll = ref(false);
+
+const selectAllIsIndeterminate = computed(() => {
+    return (apple.value && banana.value && cherry.value) !== (apple.value || banana.value || cherry.value);
+});
+
+const handleOptionChange = () => {
+    selectAll.value = apple.value && banana.value && cherry.value;
+};
+
+const handleSelectAllToggle = () => {
+    if (!(apple.value && banana.value && cherry.value)) {
+        apple.value = true;
+        banana.value = true;
+        cherry.value = true;
+    } else {
+        apple.value = false;
+        banana.value = false;
+        cherry.value = false;
+    }
+}
+
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+if ($prism) {
+    const compSource = await import('@energysage/es-ds-components/components/es-form-checkbox.vue?raw');
+    const docSource = await import('./checkbox.vue?raw');
+
+    compCode.value = $prism.normalizeCode(compSource.default);
+    docCode.value = $prism.normalizeCode(docSource.default);
+    $prism.highlight();
+}
+</script>
+
+<template>
+    <div>
+        <h1>
+            Checkbox
+        </h1>
+        <p class="mb-500">
+            Uses
+            <ds-link
+                to="https://v3.primevue.org/checkbox/"
+                target="_blank">
+                primevue checkbox
```

Comment:
PrimeVue checkbox isn't being used, should probably change this to say something like HTML input element

Reply:
🤦 done via [1fb89ec](https://github.com/EnergySage/es-ds/pull/1485/commits/1fb89eccc915ebedeb2f2804673958a79d97fb8c)